src/PageContainer.js

```diff
@@ -108,6 +135,14 @@ class PageContainer extends Component {
                         Settings
                     </div>
                 )}
+
+                {/* this only shows notification */}
+                {this.state.hover &&
+                    <NotificationPopup 
```

Comment:
nit-pick: it's a tiny bit confusing having both this and the detail popup called `Popup`, so maybe if we can only call one of them popup and the other something else? Not a big deal at all tho.